src/test/java/com/checkmarx/flow/cucumber/integration/sca_scanner/bugtrackers/ado/ScaIssuesCreationSteps.java

```diff
@@ -0,0 +1,105 @@
+package com.checkmarx.flow.cucumber.integration.sca_scanner.bugtrackers.ado;
+
+import com.checkmarx.flow.CxFlowApplication;
+import com.checkmarx.flow.config.ADOProperties;
+import com.checkmarx.flow.config.FlowProperties;
+import com.checkmarx.flow.cucumber.common.utils.JsonUtils;
+import com.checkmarx.flow.cucumber.integration.azure.publishing.AzureDevopsClient;
+import com.checkmarx.flow.dto.BugTracker;
+import com.checkmarx.flow.dto.ScanRequest;
+import com.checkmarx.flow.exception.MachinaException;
+import com.checkmarx.flow.service.ResultsService;
+import com.checkmarx.sdk.config.Constants;
+import com.checkmarx.sdk.config.ScaProperties;
+import com.checkmarx.sdk.dto.ScanResults;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import lombok.RequiredArgsConstructor;
+import org.junit.Assert;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.io.ClassPathResource;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+
+@SpringBootTest(classes = {CxFlowApplication.class, AzureDevopsClient.class})
+@RequiredArgsConstructor
+public class ScaIssuesCreationSteps {
+
+    private static final String INPUT_BASE_PATH = "cucumber/data/sample-sca-results/";
```

Comment:
"cucumber/data" is already in com.checkmarx.flow.cucumber.common.Constants#CUCUMBER_DATA_DIR.

Reply:
Done.